pilot/pkg/model/service.go

```diff
@@ -304,6 +304,10 @@ type ServiceDiscovery interface {
 	// determine the intended destination of a connection without a Host header on the request.
 	GetProxyServiceInstances(Proxy) ([]*ServiceInstance, error)
 
+	// GetProxyLabels retrieves labels attached to the sidecar used to selectively
```

Comment:
Doesn't GetProxyServiceInstances also returns labels ? 

Note we are _not_ planning to use labels in EDS in 0.8.

Reply:
Ugh, then what exactly is source label match in the API for?